sdcm/utils/docker_remote.py

```diff
@@ -103,3 +103,10 @@ def restart(self):
 
     def __str__(self):
         return f'RemoteDocker [{self.image_name}] on [{self.node}]'
+
+    @staticmethod
+    @cache
```

Comment:
I don't think so, cache is based on arguments, so when we use different node, then it will be executed.
I'll try that locally to confirm my understanding and change behavior if needed.

Reply:
Hm, double-checked it. So, yes, you are right. It does respect arguments.